cmd/experimental/posix/main.go

```diff
@@ -168,7 +169,7 @@ func newStorage(ctx context.Context, signer note.Signer) (*storage.CTStorage, er
 		return nil, fmt.Errorf("failed to initialize GCP issuer storage: %v", err)
 	}
 
-	return storage.NewCTStorage(ctx, appender, issuerStorage, reader, *enablePublicationAwaiter)
+	return storage.NewCTStorage(ctx, appender, issuerStorage, reader, *enablePublicationAwaiter, *pushbackMaxDupesInFlight)
```

Comment:
Feels like this is getting close to warranting a `CTStorageOptions` struct

Reply:
+1 (it's getting longer and longer)

Reply:
Done. We now have to pass a struct to a function that itself create a new struct... We should be able to restructure the code a bit later, but let's leave it like this for now to see what we'll need to add / remove.